app/Http/Controllers/Chat/ChatController.php

```diff
@@ -257,22 +257,28 @@ public function presence()
     public function newConversation()
     {
         $params = request()->all();
+        $target = User::lookup(get_int($params['target_id'] ?? null), 'id');
 
+        /** @var Message $message */
         $message = Chat::sendPrivateMessage(
             auth()->user(),
-            get_int($params['target_id'] ?? null),
+            $target,
             presence($params['message'] ?? null),
             get_bool($params['is_action'] ?? null)
         );
 
+        $channelJson = json_item($message->channel, 'Chat\Channel', ['first_message_id', 'last_message_id', 'users']);
+        $channelJson['icon'] = $target->user_avatar;
+        $channelJson['name'] = $target->username;
+
         return [
-            'new_channel_id' => $message->channel_id,
+            'channel' => $channelJson,
             'message' => json_item(
                 $message,
                 'Chat/Message',
                 ['sender']
             ),
-            'presence' => self::presence(),
+            'new_channel_id' => $message->channel_id,
```

Comment:
This PR intentionally doesn't doesn't require changes to lazer at the moment since the exact response hasn't been finalized yet and more changes will be coming; `new_channel_id` is currently in use, whereas `presence` isn't.